python/pyplugin_installer/qgsplugininstallerfetchingdialog.py

```diff
@@ -55,11 +55,11 @@ def __init__(self, parent):
         repositories.anythingChanged.connect(self.displayState)
 
         settings = QgsSettings()
-        self.restoreGeometry(settings.value("/Qgis/plugin-installer/fetching_geometry", QByteArray()))
+        self.restoreGeometry(settings.value("Windows/PluginInstallerFetchingDialog/geometry", QByteArray()))
```

Comment:
Would you consider using a `QgsSettins::Section` here? 
We can add more sections if needed: http://qgis.org/api/classQgsSettings.html#aba2f174ee000905cac8aff26665ed7c7

Reply:
@elpaso it's just one of over hundred entries (mostly geometries) in the **[Windows]** group. There are also dozens of entries in the **[UI]** group. At the beginning [Windows] was designed for geometries, and [UI] for other settings I guess, but now they are a little mixed. 

Do we want to port both groups to the new `QgsSettings::Section::Gui` (for now only QgsLocator uses it) or to keep a separate group for windows geometries? Unless you have a strong opinion, I'd leave it in the [Windows] for now, so later we can grep them all and port to a proper QgsSettings::Section in one move.

Reply:
No strong opinions, here, I just would like to see qgis settings slowly but steadily moving to something more stuctured and ordered, to make is easier for us developers and for users to understand in which section a particular setting can be found, the current situation is a bit messy. 
I think we need a `Windows` section in `QgsSettings`, there is already a `Gui` that was meant for GUI settings (for instance to remember the last opened tab in a dialog etc.).

Reply:
Ok, so I'll add the `Windows` section and port Installer and Manager there later today.

Reply:
@elpaso finally I didn't add the Section::Windows, as I realized the related settings were dead anyway.
Let's do it later in one move if/when porting existing geometry keys.